Submitted By: Fernando de Oliveira <famobr at yahoo dot com dot br>
Date: 2013-07-17
Initial Package Version: 2.4.1
Upstream Status: Not applicable
Origin: DJ Lucas <dj AT linuxfromscratch DOT org> (fo just added FSFrame.java)
Description: excludes known to fail tests from jtreg.

diff -Naur icedtea-2.4.1.orig/test/jtreg/excludelist.jdk.jtx icedtea-2.4.1/test/jtreg/excludelist.jdk.jtx
--- icedtea-2.4.1.orig/test/jtreg/excludelist.jdk.jtx	2013-07-05 16:56:08.448185601 -0300
+++ icedtea-2.4.1/test/jtreg/excludelist.jdk.jtx	2013-07-17 15:34:58.523131502 -0300
@@ -1,3 +1,6 @@
+# Hung the tests
+java/awt/FullScreen/SetFSWindow/FSFrame.java
+
 # Fail to compile
 javax/imageio/metadata/DOML3Node.java
 javax/swing/border/Test4856008.java
@@ -116,6 +119,7 @@
 java/awt/event/MouseWheelEvent/InfiniteRecursion/InfiniteRecursion_3.html
 java/awt/event/MouseWheelEvent/InfiniteRecursion/InfiniteRecursion_4.java
 java/awt/event/MouseWheelEvent/InfiniteRecursion/InfiniteRecursion.java
+java/awt/event/MouseWheelEvent/DisabledComponent/DisabledComponent.java
 java/awt/EventQueue/6638195/bug6638195.java
 java/awt/FileDialog/FilenameFilterTest/FilenameFilterTest.html
 java/awt/Focus/ActualFocusedWindowTest/ActualFocusedWindowBlockingTest.java
@@ -203,3 +207,208 @@
 # of build process (this class has questionable licence).
 java/util/Locale/LocaleEnhanceTest.java
 
+# NoClassDefFoundError: -classpath is set incorrectly in the testcase
+com/sun/net/httpserver/Test2.java
+com/sun/net/httpserver/bugs/B6341616.java
+java/net/Authenticator/B6870935.java
+java/net/Authenticator/B4722333.java
+java/net/Authenticator/BasicTest.java
+java/net/Authenticator/Deadlock.java
+java/net/Authenticator/B4921848.java
+java/net/Authenticator/B4962064.java
+java/net/Authenticator/BasicTest4.java
+java/net/Authenticator/AuthNPETest.java
+java/net/Authenticator/B4678055.java
+java/net/Authenticator/B4933582.sh
+java/net/Authenticator/B4759514.java
+java/net/Authenticator/BasicTest3.java
+java/net/Authenticator/B4769350.java
+java/nio/channels/SocketChannel/ExceptionTranslation.java
+java/nio/channels/ServerSocketChannel/NonBlockingAccept.java
+java/nio/channels/etc/NetworkChannelTests.java
+java/nio/channels/AsynchronousSocketChannel/Basic.java
+java/nio/channels/AsynchronousServerSocketChannel/Basic.java
+sun/net/www/AuthHeaderTest.java
+sun/net/www/protocol/http/DigestTest.java
+sun/net/www/protocol/http/BasicLongCredentials.java
+sun/net/www/protocol/http/NTLMTest.java
+sun/security/ssl/sun/net/www/protocol/https/HttpsURLConnection/HttpsProxyStackOverflow.java
+sun/security/ssl/sun/net/www/protocol/https/HttpsURLConnection/PostThruProxyWithAuth.sh
+sun/security/krb5/auto/HttpNegotiateServer.java
+
+# Win32 failure only, exits 0 unexpectedly:
+java/awt/Choice/ChoiceMouseWheelTest/ChoiceMouseWheelTest.java
+
+# robot failure (window not anchored in TWM):
+java/awt/EventDispatchThread/LoopRobustness/LoopRobustness.java
+java/awt/Focus/CloseDialogActivateOwnerTest/CloseDialogActivateOwnerTest.java
+
+# Xorg behavior change, test successfully fails to render invalid screen, but exit is unexpected
+java/awt/GraphicsDevice/CloneConfigsTest.java
+
+# Tests fail unexpectedly if no printers defined
+java/awt/print/PrinterJob/PrtException.java
+javax/print/attribute/AttributeTest.java
+javax/print/attribute/GetCopiesSupported.java
+javax/print/attribute/SidesPageRangesTest.java
+javax/print/attribute/SupportedPrintableAreas.java
+javax/print/CheckDupFlavor.java
+
+# New Xorg or evdev? Both fail on simulated button 9 press
+java/awt/Robot/AcceptExtraMouseButtons/AcceptExtraMouseButtons.java
+java/awt/Toolkit/ToolkitPropertyTest/ToolkitPropertyTest_Enable.java
+
+# This test looks to be WM dependent, window was disposed causing error
+java/awt/Window/Grab/GrabTest.java
+# Same here: Transparent frame is not transparent!
+javax/swing/JComponent/6683775/bug6683775.java
+# Anchor?
+java/awt/Focus/FocusOwnerFrameOnClick/FocusOwnerFrameOnClick.java
+
+# Timed out (possibly -samevm issue)
+java/awt/dnd/URIListBetweenJVMsTest/URIListBetweenJVMsTest.html
+java/awt/Window/WindowType/WindowType.java
+java/awt/WMSpecificTests/Mutter/MutterMaximizeTest.java
+java/beans/EventHandler/Test6179222.java
+java/beans/EventHandler/Test6277266.java
+java/beans/EventHandler/Test6788531.java
+
+# Test arguments invalid
+java/lang/invoke/ClassValueTest.java
+java/lang/invoke/InvokeGenericTest.java
+java/lang/invoke/JavaDocExamplesTest.java
+java/lang/invoke/MethodHandlesTest.java
+java/lang/invoke/MethodTypeTest.java
+java/lang/invoke/PermuteArgsTest.java
+java/lang/invoke/RicochetTest.java
+java/nio/file/Files/probeContentType/ForceLoad.java
+
+# Trust store not found (invalid argument)
+java/net/URL/TestHttps.java
+
+## Bad tests (I think, reason given before each)
+# Unexpected file type in tree
+java/nio/file/Files/probeContentType/Basic.java
+# Test passes then fails due to "exception: no exception"?!?!
+java/security/KeyPairGenerator/Failover.java
+# new/old jpeg library?
+javax/imageio/plugins/shared/BitDepth.java
+# Access denied error
+javax/management/openmbean/TabularDataOrderTest.java
+javax/swing/JFileChooser/6738668/bug6738668.java
+# Using newer version than expected (1.8.5 when 1.8 is expected)
+javax/script/VersionTest.java
+# Tests cause infinite loop (1.8.5 changes?)
+javax/swing/JEditorPane/6917744/bug6917744.java
+javax/swing/JInternalFrame/Test6325652.java
+javax/swing/JInternalFrame/Test6505027.java
+javax/swing/JInternalFrame/Test6802868.java
+javax/swing/JLabel/6596966/bug6596966.java
+javax/swing/JLabel/7004134/bug7004134.java
+javax/swing/JLayer/6824395/bug6824395.java
+javax/swing/JList/6462008/bug6462008.java
+javax/swing/JOptionPane/7138665/bug7138665.java
+javax/swing/JPopupMenu/4966112/bug4966112.java
+javax/swing/JPopupMenu/6495920/bug6495920.java
+javax/swing/JPopupMenu/6827786/bug6827786.java
+javax/swing/JPopupMenu/6987844/bug6987844.java
+javax/swing/JScrollBar/4708809/bug4708809.java
+javax/swing/JScrollBar/4865918/bug4865918.java
+javax/swing/JScrollBar/6542335/bug6542335.java
+javax/swing/JScrollPane/Test6526631.java
+javax/swing/JSlider/6348946/bug6348946.java
+javax/swing/JSlider/6848475/bug6848475.java
+javax/swing/JSplitPane/4885629/bug4885629.java
+javax/swing/JTabbedPane/4624207/bug4624207.java
+javax/swing/JTable/6263446/bug6263446.java
+javax/swing/JTable/6777378/bug6777378.java
+javax/swing/JTableHeader/6884066/bug6884066.java
+javax/swing/JTableHeader/6889007/bug6889007.java
+javax/swing/JTextArea/4697612/bug4697612.java
+javax/swing/JTree/6263446/bug6263446.java
+javax/swing/JTree/6505523/bug6505523.java
+javax/swing/JViewport/7107099/bug7107099.java
+javax/swing/plaf/basic/BasicHTML/4251579/bug4251579.java
+javax/swing/PopupFactory/6276087/NonOpaquePopupMenuTest.java
+javax/swing/Security/6657138/ComponentTest.java
+javax/swing/SwingUtilities/4917669/bug4917669.java
+javax/swing/text/CSSBorder/6796710/bug6796710.java
+javax/swing/text/DefaultEditorKit/4278839/bug4278839.java
+
+# Still using LCMS1 profile (LCMS2 tests need backported from java 8 tree)
+java/awt/image/ImagingOpsNoExceptionsTest/ImagingOpsNoExceptionsTest.java
+sun/java2d/cmm/ColorConvertOp/MTSafetyTest.java
+sun/java2d/cmm/ProfileOp/ReadWriteProfileTest.java
+
+# Invalid test case of initializer (constant)
+sun/net/idn/TestStringPrep.java
+
+# Fails to compile
+sun/security/ec/TestEC.java
+
+# http://icedtea.classpath.org/bugzilla/show_bug.cgi?id=914
+com/oracle/security/ucrypto/TestAES.java
+com/oracle/security/ucrypto/TestRSA.java
+
+# http://icedtea.classpath.org/bugzilla/show_bug.cgi?id=913
+com/oracle/security/ucrypto/TestDigest.java
+
+# Suspected WM behavior issues
+java/awt/EventDispatchThread/HandleExceptionOnEDT/HandleExceptionOnEDT.java
+java/awt/EventDispatchThread/LoopRobustness/LoopRobustness.html
+java/awt/font/TextLayout/DecorationBoundsTest.java
+java/awt/font/TextLayout/UnderlinePositionTest.java
+java/awt/Frame/ShapeNotSetSometimes/ShapeNotSetSometimes.java
+java/awt/FullScreen/TranslucentWindow/TranslucentWindow.java
+javax/swing/JComponent/4337267/bug4337267.java
+
+# Infinate Loop
+javax/swing/text/JTextComponent/5074573/bug5074573.java
+
+# nspr.so alread loaded
+sun/security/krb5/runNameEquals.sh
+sun/security/pkcs11/Cipher/TestPKCS5PaddingError.java
+sun/security/pkcs11/Cipher/TestRawRSACipher.java
+sun/security/pkcs11/Cipher/TestRSACipher.java
+sun/security/pkcs11/Cipher/TestSymmCiphers.java
+sun/security/pkcs11/Cipher/TestSymmCiphersNoPad.java
+sun/security/pkcs11/ec/ReadCertificates.java
+sun/security/pkcs11/ec/ReadPKCS12.java
+sun/security/pkcs11/ec/TestCurves.java
+sun/security/pkcs11/ec/TestECDH.java
+sun/security/pkcs11/ec/TestECDSA.java
+sun/security/pkcs11/ec/TestECGenSpec.java
+sun/security/pkcs11/ec/TestKeyFactory.java
+sun/security/pkcs11/fips/TrustManagerTest.java
+sun/security/pkcs11/KeyAgreement/TestShort.java
+sun/security/pkcs11/KeyGenerator/DESParity.java
+sun/security/pkcs11/KeyGenerator/TestKeyGenerator.java
+sun/security/pkcs11/KeyStore/ClientAuth.sh
+sun/security/pkcs11/MessageDigest/ByteBuffers.java
+sun/security/pkcs11/MessageDigest/DigestKAT.java
+sun/security/pkcs11/MessageDigest/ReinitDigest.java
+sun/security/pkcs11/Provider/ConfigShortPath.java
+sun/security/pkcs11/rsa/TestCACerts.java
+sun/security/pkcs11/rsa/TestKeyFactory.java
+sun/security/pkcs11/Secmod/AddPrivateKey.java
+sun/security/pkcs11/Secmod/AddTrustedCert.java
+sun/security/pkcs11/Secmod/Crypto.java
+sun/security/pkcs11/Secmod/GetPrivateKey.java
+sun/security/pkcs11/Secmod/JksSetPrivateKey.java
+sun/security/pkcs11/Secmod/TrustAnchors.java
+sun/security/pkcs11/SecureRandom/Basic.java
+sun/security/pkcs11/SecureRandom/TestDeserialization.java
+sun/security/pkcs11/Serialize/SerializeProvider.java
+sun/security/pkcs11/Signature/ByteBuffers.java
+sun/security/pkcs11/Signature/ReinitSignature.java
+sun/security/pkcs11/Signature/TestDSA.java
+sun/security/pkcs11/Signature/TestRSAKeyLength.java
+sun/security/pkcs11/sslecc/ClientJSSEServerJSSE.java
+sun/security/pkcs11/tls/TestKeyMaterial.java
+sun/security/pkcs11/tls/TestMasterSecret.java
+sun/security/pkcs11/tls/TestPremaster.java
+sun/security/pkcs11/tls/TestPRF.java
+sun/security/pkcs11/SampleTest.java
+sun/security/ssl/com/sun/net/ssl/internal/ssl/EngineArgs/DebugReportsOneExtraByte.sh
+sun/security/tools/jarsigner/ec.sh
+
diff -Naur icedtea-2.4.1.orig/test/jtreg/excludelist.langtools.jtx icedtea-2.4.1/test/jtreg/excludelist.langtools.jtx
--- icedtea-2.4.1.orig/test/jtreg/excludelist.langtools.jtx	2013-07-05 16:56:08.448185601 -0300
+++ icedtea-2.4.1/test/jtreg/excludelist.langtools.jtx	2013-07-17 15:34:03.275322332 -0300
@@ -27,3 +27,7 @@
 
 # `main' threw exception: java.lang.Error: 64/256 test cases passed
 tools/javap/OptionTest.java
+
+# Fails to compile
+tools/javac/processing/6499119/ClassProcessor.java
+
